### DISCUSSION TITLE
About Pro lable<title>

### DISCUSSION THREAD

**Discussion by saurabhpawar1997** [community/community#49269] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

How to get the pro label on my GitHub profile?

**Comment by queenofcorgis** [comment, marked as answer]
Hi there! The Pro label is for those subscribed to our Pro plan - for more details on what this plan entails - check out our [docs](https://docs.github.com/en/get-started/learning-about-github/githubs-products#github-pro)!
